docs/manual/cli/db-backups.en.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Database Backups"
+description: "Creating database backups made easy"
+aliases:
+    - /en/cli/database-backups/
+---
+
+{{< version "4.13" >}}
+
+Contao provides everything you need for a reliable database backup. Once configured, you can enjoy your good night
+sleep, because whatever happens to the database, you have your backups ready and can restore the latest one
+with just a single command!
+
+By default, the backups are stored and managed in `var/backups`. The following commands are available
+for managing them:
+
+## contao:backup:create
+
+Of course, the most important command. Without having a backup, we can do very little with the other commands. 
+The simplest form of use is without any arguments:
+
+```bash
+php vendor/bin/contao-console contao:backup:create
+```
+
+Done. Contao will now create a backup in `var/backups` and add the current date and time to it.
+By default, it is also compressed directly to avoid wasting space on your system.
+
+Backups always start with an arbitrary name (`backup` by default) and are separated from the date/time information by `__`.
+So you will have a file in `var/backups` which looks like this: `backup__20220126153243.sql.gz`.
+
+If you want, you can change the name and thus the time yourself by specifying the complete backup name. By omitting
+`.gz` you also conveniently disable compression:
+
+```bash
+php vendor/bin/contao-console contao:backup:create my_super_backup_name__20220101000000.sql
+```
+
+{{% notice tip %}}
+Every time a new backup is created, Contao automatically cleans up obsolete backups. See
+section "[Configuration](#configuration)" for more information.
+{{% /notice %}}
+
+## contao:backup:list
+
+This command lets us display the existing backups. The output should look something like this:
+
+```bash
+--------------------- ----------- ------------------------------- 
+Created               Size        Name
+--------------------- ----------- ------------------------------- 
+2022-01-26 15:32:43   73.14 KiB   backup__20220126153243.sql.gz
+--------------------- ----------- -------------------------------
+```
+
+## contao:backup:restore
+
+This command lets you restore one of the existing backups. By default, the most recent backup is restored.
+However, you can also specify a specific backup:
+
+```bash
+# The most recent backup
+php vendor/bin/contao-console contao:backup:restore
+
+# Any given backup
+php vendor/bin/contao-console contao:backup:restore backup__20220126153243.sql.gz
+```
+
+{{% notice warning %}}
+The database must be completely emptied before restoring a backup.
+{{% /notice %}}
```

Comment:
The SQL backups contain `DROP TABLE IF EXISTS`, so that's not actually necessary - or does the restore command check if the database is not empty beforehand?

Reply:
Ah, I completely forgot about that although I wrote it myself 🤣